### PR TITLE
Release/2.1.0

### PR DESCRIPTION
**Pull request by thoma202**
Hi, 

This pull requests fixes google maps by using controller's AddJS to import google maps 
Also this version does not allow to install the module if curl or soap is not available.

### CODE REVIEW

File: js/mondialrelay.js [hunk 1]
```diff
@@ -1127,7 +1127,7 @@ var PS_MRObject = (function($, undefined) {
 					}
 					
 					
-					if($(e).attr('checked') == 'checked') {
+					if($(e).attr('checked') == 'checked' || $(e).attr('checked')) {
```
Comment (Quetzacoalt91) [inline js/mondialrelay.js]: Why do you write the same test twice ?

Comment (thoma202) [inline js/mondialrelay.js]: Good question, This was not supposed to be like this :).
Let me double check

Comment (thoma202) [inline js/mondialrelay.js]: Oh no, this is actually two different tests. 
Depending on jquery version, a checked checkbox will send back 'checked' or true :)

Comment (Quetzacoalt91) [inline js/mondialrelay.js]: Indeed, I read multiple times this test but did not see that point.

Comment (thoma202) [inline js/mondialrelay.js]: So it's ok for you ? :)

**Comment by Quetzacoalt91** [comment]
Yes this is good for me, that's why the PR has just been merged. :)

Thanks for your contribution.

Best regards

**Comment by thoma202** [comment]
Thanks :+1:

**Pull request merged**